ALUM/ALUM/Components/Tag.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Tag.swift
+//  ALUM
+//
+//  Created by Jenny Mar on 1/28/23.
+//
+
+import SwiftUI
+
+struct Tag: View {
+
+    @State var text: String = ""
+    @State var isBlue: Bool = true
+    @State var isShowing: Bool = true
+
+    var body: some View {
```

Comment:
Can we just split this Tag component into two components? TagDisplay and AddTagButton. 

The only reason we would combine both tags would be if we had reused code but here we don't have any logic common between the two components.